src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -299,19 +345,19 @@ function PayButton ({ showConfirmationModal, transaction }) {
   </Button>
 }
 
-function RejectButton ({ showConfirmationModal, transaction }) {
-  const action = 'decline'
+function DeclineOrCancelButton ({ showConfirmationModal, transaction, isDeclined }) {
+  const action = isDeclined ? 'cancel' : 'decline'
   return <Button
     onClick={() => showConfirmationModal(transaction, action)}
     styleName='reject-button'>
-    <p>Decline</p>
+    <p>{isDeclined ? 'Cancel' : 'Decline'}</p>
   </Button>
 }
 
-function NewTransactionModal ({ handleClose, toggleModal }) {
+function NewTransactionModal ({ handleClose, isNewTransactionModalVisible }) {
   return <Modal
     contentLabel={'Create a new transaction.'}
-    isOpen={!!toggleModal}
+    isOpen={!!isNewTransactionModalVisible}
```

Comment:
this is close to nitpicking, but because `isNewTransactionModalVisible` is a bool, `!!` is redundant.

`isNewTransactionModalVisible === !!isNewTransactionModalVisible` for all values of `isNewTransactionModalVisible`

Reply:
Good point. :+1:  I was using the previous modal code as boilerplate, and just didn't make that update update after we pegged the isOpen value to a boolean.  This is a good update, even though it's small.